straxen/test_utils.py

```diff
@@ -16,58 +14,6 @@
 
 
 nt_test_run_id = '012882'
-test_run_id_1T = '180423_1021'
-
-testing_config_1T = dict(
-    hev_gain_model=('1T_to_pe_placeholder', False),
-    gain_model=('1T_to_pe_placeholder', False),
-    elife=1e6,
-    electron_drift_velocity=1e-4,
-    electron_drift_time_gate=1700,
-)
-
-# Let's make a dummy map for NVeto
-_nveto_pmt_dummy = {'channel': list(range(2000, 2120)),
-                    'x': list(range(120)),
-                    'y': list(range(120)),
-                    'z': list(range(120)),
-                    }
-_nveto_pmt_dummy_df = DataFrame(_nveto_pmt_dummy)
-
-# Some configs are better obtained from the strax_auxiliary_files repo.
-# Let's use small files, we don't want to spend a lot of time downloading
-# some file.
-_testing_config_nT = dict(
-    nn_architecture=
-    aux_repo + 'f0df03e1f45b5bdd9be364c5caefdaf3c74e044e/fax_files/mlp_model.json',
-    nn_weights=
-    aux_repo + 'f0df03e1f45b5bdd9be364c5caefdaf3c74e044e/fax_files/mlp_model.h5',
-    gain_model=("to_pe_placeholder", True),
-    elife=1e6,
-    baseline_samples_nv=10,
-    fdc_map=pax_file('XENON1T_FDC_SR0_data_driven_3d_correction_tf_nn_v0.json.gz'),
-    gain_model_nv=("adc_nv", True),
-    gain_model_mv=("adc_mv", True),
-    nveto_pmt_position_map=_nveto_pmt_dummy,
-    s1_xyz_map=f'itp_map://resource://{pax_file("XENON1T_s1_xyz_lce_true_kr83m_SR1_pax-680_fdc-3d_v0.json")}?fmt=json',
-    s2_xy_map=f'itp_map://resource://{pax_file("XENON1T_s2_xy_ly_SR1_v2.2.json")}?fmt=json',
-    electron_drift_velocity=1e-4,
-    s1_aft_map=f'itp_map://resource://{aux_repo + "023cb8caf2008b289664b0fefc36b1cebb45bbe4/strax_files/s1_aft_UNITY_xyz_XENONnT.json"}?fmt=json', # noqa
-    s2_optical_map=aux_repo + '9891ee7a52fa00e541480c45ab7a1c9a72fcffcc/strax_files/XENONnT_s2_xy_unity_patterns.json.gz',  # noqa
-    s1_optical_map=aux_repo + '9891ee7a52fa00e541480c45ab7a1c9a72fcffcc/strax_files/XENONnT_s1_xyz_unity_patterns.json.gz',  # noqa
-    electron_drift_time_gate=2700,
-    hit_min_amplitude='pmt_commissioning_initial',
-    hit_min_amplitude_nv=20,
-    hit_min_amplitude_mv=80,
-    hit_min_amplitude_he='pmt_commissioning_initial_he',
-    avg_se_gain=1.0,
-    se_gain=1.0,
-    rel_extraction_eff=1.0,
-    rel_light_yield=1.0,
-    g1=0.1,
-    g2=10,
-    bayes_config_file=f'resource://{aux_repo +"2df37896923eb9ca3157befb6274cb697ac2f4f7/strax_files/dummy_bayes_model.npz"}?fmt=npy',
-)
```

Comment:
No more testing with very unclear configs / infinite re-iterations to make the test works 🎉